src/App.tsx

```diff
@@ -1,14 +1,77 @@
 /* eslint-disable max-len */
-import React from 'react';
+
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos, getUser } from './api';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [user, setUser] = useState<User | null>(null);
+  const [filteredField, setFilteredField] = useState<string>('All');
+  const [query, setQuery] = useState<string>('');
+
+  const preparedTodos = todos.filter(todo => {
+    if (filteredField === 'completed') {
+      return todo.completed === true;
+    }
+
+    if (filteredField === 'active') {
+      return todo.completed === false;
+    }
```

Comment:
Recommendation: 
Consider using Enum for limited options eg. 
Enum Field {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
}
And replace 'All' with Field.All

Reply:
> Recommendation: Consider using Enum for limited options eg. Enum Field { All = 'all', Active = 'active', Completed = 'completed', } And replace 'All' with Field.All

Good point
And use switch case instead